extensions/amp-loader/0.1/amp-loader.js

```diff
@@ -257,14 +264,52 @@ class LoaderBuilder {
     `;
   }
 
+  /**
+   * `<amp-ad>s` have several different classes, so putting the code here for
+   * now since it is the safest way to make sure that they all get the correct
+   * loader.
+   *
+   * Since the implementation may have a delay before loading, we would need to
+   * make sure the ads loader is present, even if the implementation has not
+   * yet downloaded.
+   *
+   * TODO(sparhami) Move this out of amp-loader into something common for ads.
+   * @return {!Element}
+   * @private
+   */
+  getAdsLogo_() {
+    const html = htmlFor(this.element_);
+    return html`
+      <svg viewBox="0 0 72 72">
+        <rect
+          fill="none"
+          stroke="currentColor"
+          x="26.5"
```

Comment:
Any way to make this look better? Maybe a percentage x, y coordinate combined with a negative transform?

Reply:
Re-implemented this using just regular DOM, which also makes sure that it is actually centered.